packages/atomic-react/src/components/commerce/CommerceInterfaceWrapper.tsx

```diff
@@ -2,19 +2,19 @@ import type {JSX, i18n} from '@coveo/atomic';
 import React, {useEffect, useRef} from 'react';
 import {AtomicCommerceInterface} from '../stencil-generated/commerce';
 
-type ExecuteSearch = HTMLAtomicCommerceInterfaceElement['executeFirstSearch'];
+type ExecuteRequest = HTMLAtomicCommerceInterfaceElement['executeFirstRequest'];
 
 /**
  * The properties of the AtomicCommerceInterface component
  */
 interface WrapperProps
   extends Omit<JSX.AtomicCommerceInterface, 'i18n' | 'pipeline' | 'searchHub'> {
   /**
-   * An optional callback function that can be used to control the execution of the first query.
+   * An optional callback function that can be used to control the execution of the first request.
    *
-   * If not provided, a default function will be used, which execute the first query immediately after initialization.
+   * If not provided, a default function will be used, which execute the request query immediately after initialization.
```

Comment:
```suggestion
   * If not provided, a default function will be used, which execute the request immediately after initialization.
```

Reply:
Fixed to
> If not provided, a default function will be used, which executes the first request immediately after initialization.

in https://github.com/coveo/ui-kit/pull/4095/commits/fe9ac79cf66175479b331eae7330394cbdd7e586